### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by boojack**

### CODE REVIEW

**Comment by boojack** [comment]
PTAL

File: README.md [hunk 1]
```diff
@@ -13,73 +10,66 @@
...
-- 2016-6-30: Alert to notie
+  ```shell
+  yarn dev
+  ```
 
-- 2016-6-28: Add clear btn
+  The website will be served at http://localhost:3000.
 
-- 2016-6-28: Better view for "many star" repos (use current star number as the last point on the graph)
+- **Chrome extension** supports the basic chart viewer as a free additional product.
```
Comment (tianzhou) [inline README.md:60]: I still suggest to put chrome extension on the top, or at least mention it with a #link pointing to this section

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by boojack** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**